### PR TITLE
Update documented Maven requirement: 3.5.3 -> 3.5.4

### PR DESCRIPTION
**Pull request by edh-oss**
### Proposed changelog entries

* Entry 1: Update CONTRIBUTING.md to reflect current Maven version requirement

### CODE REVIEW

**Review by oleg-nenashev — APPROVED** [review]
I am not sure what changed in the last like, but indeed we target 3.5.4 now. Thanks!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This page provides information about contributing code to the Jenkins core codeb
      - In Jenkins project we usually use [OpenJDK],
   but you can use other JDKs as well.
      - Java 9+ is **not supported** in Jenkins.
```
Comment (daniel-beck) [inline CONTRIBUTING.md]: While we're at it… @oleg-nenashev Time to clean this up?

**Review by daniel-beck — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (oleg-nenashev) [reply CONTRIBUTING.md]: It is

**Comment by oleg-nenashev** [comment]
Thanks @edh-oss !

**Pull request merged**